platform/src/main/java/org/stellar/anchor/platform/payment/config/CirclePaymentConfig.java

```diff
@@ -0,0 +1,9 @@
+package org.stellar.anchor.platform.payment.config;
+
+import lombok.Data;
+
+@Data
+public class CirclePaymentConfig {
```

Comment:
I don't think we need this at all anymore, but could be worth to keep it just in case we need to revert this in the future 🤔